content/account/sso.textile

```diff
@@ -0,0 +1,97 @@
+---
+title: Single sign-on (SSO)
+meta_description: "Single sign-on enables users to authenticate with Ably using your own identity provider."
+meta_keywords: "sso, saml, strict mode, single sign-on"
+---
+
+Single sign-on (SSO) enables your users to authenticate via any SAML-compatible identity provider.
+
+<aside date-type='note'>
+<p>User provisioning and deprovisioning via SCIM or SAML JIT is not supported.</p>
```

Comment:
maybe `is not currently supported`? To hint that we may build this in the future?